watson_developer_cloud/conversation_v1.py

```diff
@@ -30,6 +30,68 @@ def __init__(self, version, url=default_url, **kwargs):
                                              **kwargs)
         self.version = version
 
+    def list_workspaces(self):
+        """
+        List workspaces available.
+        This includes pagination info.
+        """
+        params = {'version': self.version}
+        return self.request(method='GET',
+                            url='/v1/workspaces',
+                            params=params,
+                            accept_json=True)
+
+    def get_workspace(self, workspace_id, export=False):
+        """
+        Get a specific workspace
+        :param: workspace_id  the guid of the workspace
+        :param: export (optional) return all workspace data
+        """
+        params = {'version': self.version}
+        if export:
+            params['export'] = True
+
+        return self.request(method='GET',
+                            url='/v1/workspaces/{0}'.format(workspace_id),
+                            params=params,
+                            accept_json=True)
+
+    def delete_workspace(self, workspace_id):
+        """
+        Deletes a given workspace.
+        :param: workspace_id the guid of the workspace_id
+        """
+        params = {'version': self.version}
+        return self.request(method='DELETE',
+                            url='/v1/workspaces/{0}'.format(workspace_id),
+                            params=params,
+                            accept_json=True)
+
+    def create_workspace(self, workspace_data):
```

Comment:
It's, `name, description, language, metadata, counterexamples, dialog_nodes, entities, intents`? I would still make them all top-level and provide `=None` defaults for most of them.